new-frontiers-14/frontier-station-14
# Ghost Roles: fix whitelist checks

## About the PR



Restores ghost role whitelist checks.  Removed a few remaining "FRONTIER MERGE" comments with no relevance (commented code, unused imports).

## Why / Balance


Error in previous upstream merge.

## How to test


0. Build the game, should build fine.
1. Spawn with unwhitelisted account.
2. /ghost.  Check roles, emotional/station pets should be disabled.
3. Comment out GhostRoleSystem.cs:598 (removes prototype, which the UI relies on to disable buttons).
4. Build, spawn with unwhitelisted account.
5. /ghost.  Check roles, emotional/station pets should be enabled.
6. Click all available whitelisted roles.  Nothing should happen (server-side checks on Takeover, JoinRaffle).

## Media


![image](https://github.com/user-attachments/assets/abe980eb-e267-48f9-add3-a7e46d11dd10)

## Requirements


## Breaking changes


**Changelog**



:cl:
- fix: Fixed ghost role whitelist requirement checks.
